<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-20T08:55:21 EDT" name="Suite" finished-at="2024-09-20T08:55:27 EDT" duration-ms="5592">
    <groups>
    </groups>
    <test started-at="2024-09-20T08:55:21 EDT" name="Test" finished-at="2024-09-20T08:55:27 EDT" duration-ms="5592">
      <class name="api.Tests.Run_Test">
        <test-method is-config="true" signature="Data()[pri:0, instance:api.Tests.Run_Test@69e153c5]" started-at="2024-09-20T08:55:22 EDT" name="Data" finished-at="2024-09-20T08:55:22 EDT" duration-ms="354" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Data -->
        <test-method signature="createUser()[pri:1, instance:api.Tests.Run_Test@69e153c5]" started-at="2024-09-20T08:55:22 EDT" name="createUser" finished-at="2024-09-20T08:55:26 EDT" duration-ms="4154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createUser -->
        <test-method signature="findUser()[pri:2, instance:api.Tests.Run_Test@69e153c5]" started-at="2024-09-20T08:55:26 EDT" name="findUser" finished-at="2024-09-20T08:55:27 EDT" duration-ms="380" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<404> but was:<200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<404> but was:<200>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:128)
at org.junit.Assert.assertEquals(Assert.java:472)
at org.junit.Assert.assertEquals(Assert.java:456)
at api.Tests.Run_Test.findUser(Run_Test.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findUser -->
      </class> <!-- api.Tests.Run_Test -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
